### PR TITLE
Docs rsa support

### PR DESCRIPTION
**Pull request by tufanbarisyildirim**
Mentioned in docs that it supports RSA too  (https://github.com/centrifugal/centrifuge/pull/79)

### CODE REVIEW

File: internal/admin/handlers.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type Config struct {
 	// Password is an admin password.
 	Password string
 
-	// Secret is a secret to generate auth token for admin requests.
+	// TokenHMACSecretKey is a secret to generate auth token for admin requests.
```
Comment (FZambia) [inline internal/admin/handlers.go]: Looks like excess auto replacement

File: main.go [hunk 2]
```diff
@@ -771,20 +772,20 @@ func pathExists(path string) (bool, error) {
 }
 
 var jsonConfigTemplate = `{
-  "secret": "{{.Secret}}",
+  "secret": "{{.TokenHMACSecretKey}}",
```
Comment (FZambia) [inline main.go]: Won't work as anonymous struct does not have TokenHMACSecretKey field

File: main.go [hunk 3]
```diff
@@ -884,7 +885,12 @@ func nodeConfig() *centrifuge.Config {
 
 	cfg.Version = VERSION
 	cfg.Name = applicationName()
-	cfg.Secret = v.GetString("secret")
```
Comment (FZambia) [inline main.go]: Need to leave this option for backwards compatibility

**Comment by tufanbarisyildirim** [comment]
I shoulnd have push those changes. Will go over again. Ignore for now.

File: internal/admin/handlers.go (hunk 1 above)
Comment (tufanbarisyildirim) [reply internal/admin/handlers.go]: Yeah that was the first thing I did. Im on the correct path now. I will fix,please ignore.

File: config.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
-  "secret": "secret",
```
Comment (FZambia) [inline config.json]: Let's use secret for now - I will add `token_hmac_secret_key ` option in future commits

File: config.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
...
+  "token_hmac_secret_key": "secret",
+  "token_rsa_public_key": "-----BEGIN PUBLIC KEY-----\nMFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBAJ+A48YVz4XVzEIHEQAYdmBg9Y4yaNhu\nMTAfeZ9BZRaYTNNMloE0MGLII4aZukRjiW0pjT96i1PRezH7jk10HzcCAwEAAQ==\n-----END PUBLIC KEY-----",
```
Comment (FZambia) [inline config.json]: I suppose we can remove this from config example because it does not have many sense without knowing private key

**Comment by tufanbarisyildirim** [comment]
it should pass all checks when https://github.com/centrifugal/centrifuge/pull/79 merges

**Pull request closed without merging**